tests/integration/test_user_invite.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from unittest.mock import patch
+from models.user import RoleEnum
+from resources.email import Email
+
+
+@pytest.mark.usefixtures('client_class', 'empty_test_db')
+class TestUserInvite:
+    def setup(self):
+        self.endpoint = '/api/user/invite'
+
+    @patch.object(Email, 'send_user_invite_msg')
+    def test_invite_user(self, send_user_invite_msg, valid_header, user_attributes):
+        response = self.client.post(self.endpoint, headers=valid_header, json=user_attributes(role='STAFF'))
```

Comment:
I created issue codeforpdx/dwellingly-app/issues/491 to address the issue of passing a string as a value rather than the agreed upon ints. This is the first resource to use the User schema and the schema is requiring string values instead of ints.